src/interactive/lashprompt.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+from typing import Dict, Tuple
+
+import discord
+
+from interactive.promptmodal import PromptModal
+from interactive.userunique import UserUniqueView
+from utils.lookups import EMOJI_FORWARD, random_emoji
+from utils import async_context_wrap
+
+logger = logging.getLogger(__name__)
+
+
+class LashPrompt(discord.ui.View):
```

Comment:
Could we follow the naming convention that `XPrompt` becomes `XUserPrompt` and then `XGetPromptView` is just `XPrompt`?

Reply:
I notice you named them like this because we already had things like `UserUniqueView` and `GatherPlayersView`. Maybe we do a refactor PR later and rename all these things in one commit.